src/components/video-view.tsx

```diff
@@ -64,6 +70,12 @@ export const VideoView = ({
     }
   }, [percentComplete, setTargetVideoIndex, timelineMarks]);
 
+  // Show the front video when it's finished loading
```

Comment:
This comment doesn't seem quite right given what the code below is doing.

Reply:
Thanks, good catch